fixtures/cassettes/hand-edited_requests.yml

```diff
@@ -295,6 +295,19 @@ http_interactions:
   digest: invalid_but_load_page
 - request:
     method: get
+    uri: https://cmr.earthdata.nasa.gov/search/collections.json?include_facets=true&page_size=20&page_num=1&include_has_granules=true&include_granule_counts=true
```

Comment:
Yes, but it's very strange, because this is pretty much a default collections search. Some logic must be broken.